2/수박수박수박수박수?.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+  let answer = "";
+  for (let i = 1; i <= n; i++) {
+    if (i % 2 === 1 ? (answer += "수") : (answer += "박"));
```

Comment:
```suggestion
    i % 2 === 1 ? (answer += "수") : (answer += "박");
```
이미 조건문의 한 종류인 삼항연산자를 사용하셨기 때문에 if문은 넣어줄 필요가 없습니다.